cvat-ui/src/components/login-with-token/login-with-token.tsx

```diff
@@ -24,7 +26,7 @@ export default function LoginWithTokenComponent(): JSX.Element {
     );
 
     if (cookies.sessionid && cookies.csrftoken) {
-        return <Redirect to='/tasks' />;
+        return <Redirect to={redirectParam || '/tasks'} />;
```

Comment:
Redirection works fine at first time. 
Unfortunately when user is already loged in {cookies are set} It does not redirect to requested page any more

Reply:
Please, try another implementation according to updated PR description.

Reply:
Hi, Thanks for the update. 
Tested on Chrome 87 - macOS (Big Sur) and Windows 10
Unfortunatle issue mentioned before still persists:

Steps to reproduce:
1. 
Action:
Login to CVAT using ?nexto fo example  http:/<ip>/auth/login-with-token/aj52qlbiux7manf8xwkz9etz1ivg8atk/nR9CeVzrL0AZMh8ZuAEzBr6swCeWUVbWT6KfScBbNuCMUW4dU1tU50tKb7ofUp9B?next=/tasks/1
Expecteted result: 
CVAT is logged and page is redirected to /tasks/1
2.
Action:
Open new browser tab and past link once again:
Expected result:
CVAT is logged and page is redirected to **/tasks/1**
Actual result:
CVAT is logged but page is redirected to **/tasks**

Regards

Reply:
@tdowgiel 
I thought you told about login with UI elements (twice or more) with the redirection query parameter. Anyway, looks like we fixed also login with a token.